### PR TITLE
Add Chart Component for #20

### LINKED ISSUE #20: Add Chart Component
As per [Figma](https://www.figma.com/file/qOoG6pKbWKi2ldzHT2cHA6/Crypto-Component---Light-Mode-(Part-2)-(Community)?node-id=1%3A6329&t=Jrp064v41nbf7cAV-4)

### Preview

<div align="center">

Image

</div>

### Variants
- [ ] Light

### PR DESCRIPTION
**Pull request by mrxshinji**
Fixes #20 

# Preview

image

## Changes

- Added Chart component and Chart story page
- Added Link from story page to Chart story page
- Minor name link changes

### CODE REVIEW

File: src/components/chart/chart.css [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+.chart-body {
+  height: var(--h);
+  width: 20px;
+  border-radius: 3px 3px 0px 0px;
+}
+
+.body-1 {
+  background-color: #5856d6;
+}
+.body-2 {
```
Comment (aftabrehan) [inline src/components/chart/chart.css:162]: Just a minor fix for a good code.

```suggestion
}

.body-2 {
```

File: src/routes/story-page/story-page.jsx [hunk 2]
```diff
@@ -8,11 +8,12 @@ const StoryPage = () => {
       <Link to={`/`}>Back to home</Link>
       <Link to={`/stories/component`}>A-component</Link>
       <Link to={`/stories/button`}>Example Button</Link>
-      <Link to={`/stories/quicktransfer`}>Quick transfer component</Link>
-      <Link to={`/stories/notification`}>Notification Bar Component</Link>
+      <Link to={`/stories/quicktransfer`}>Quick transfer</Link>
+      <Link to={`/stories/notification`}>Notification Bar</Link>
```
Comment (aftabrehan) [inline src/routes/story-page/story-page.jsx:12]: @mrxshinji 👍

File: src/routes/chart-page/index.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import Chart from '../../components/chart/chart'
```
Comment (aftabrehan) [inline src/routes/chart-page/index.jsx:1]: Could you configure the base path in `jsconfig.json`? If you can then raise a separate PR for it, also make sure to create an issue for it.

```suggestion
import Chart from 'components/chart/chart'
```

**Review by aftabrehan — APPROVED** [review]
Looks good to me! 👍 I'm approving this PR, just a minor fix and then you're good to go 🚀

File: src/routes/chart-page/index.jsx (hunk 3 above)
Comment (mrxshinji) [reply src/routes/chart-page/index.jsx:1]: Never done that before, so no ..... 
And there is multiple folder (component/routes/asset) we need to access though....

Comment (aftabrehan) [reply src/routes/chart-page/index.jsx:1]: I'll take care of it. Thanks 👉

**Pull request merged**